example/app.py

```
@@ -37,5 +48,266 @@
+        app.stress_leak = []
+    
+    leak_bytes = leak_mb * 1024 * 1024
+    leak_data = array('B', [random.randint(0, 255) for _ in range(leak_bytes)])
+    app.stress_leak.append(leak_data)
+    results.append(f"Leak: {leak_mb}MB added")
+    
+    # 4. Clean up memory stress
+    try:
+        del large_array
+    except:
+        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)